### ISSUE TITLE
Pairing Sensor Tag CC2650 with Windows 8.1/10 failure

### ISSUE THREAD

**Issue by NNM11** [clovett/SensorTag-for-Windows#1]
Hi,

I tried to pair the sensor tag CC2650 with both Windows 8.1 and Windows 10 notebooks and neither one could successfully pair with the sensor tag.  Is there something I need to configure to make this work?

Thanks,

**Comment by clovett** [comment]
interesting, I’ve had no problems with the older Sensor Tag CC2541.  I wonder what has changed?  I’ll have to get a device and see….






Sent from Windows Mail





From: NNM11
Sent: ‎Wednesday‎, ‎July‎ ‎8‎, ‎2015 ‎4‎:‎53‎ ‎PM
To: clovett/SensorTag-for-Windows





Hi,

I tried to pair the sensor tag CC2650 with both Windows 8.1 and Windows 10 notebooks and neither one could successfully pair with the sensor tag. Is there something I need to configure to make this work?

Thanks,

**Comment by j0hnnyk** [comment]
I believe you will have to update the firmware to version 1.12 (Jun 23 2015) in order to pair with windows 8.1 and Windows 10. I used the free TI app to load the firmware OAD. The app only runs on Android and iOS, of course...  :(  I used a nexus 7.

That being said, I am only able to get Temp, Humidity, and simple key sensors working properly. I am not having much luck with accelerometer, gyro, lux, pressure...

**Comment by clovett** [comment]
I have confirmed that a firmware upgrade fixes the problem with the new sensor tag.  But I have some work to do to update the software to fully understand the new sensor characteristics.  There are some changes to the GATT profile.  I checked in a new version that prepares for this by adding version numbers to the GATT services.  The easiest way to get the new firmware is to order the SimpleLink SensorTag DevPack ($22) and install the SmartRF Flash Programmer 2 which comes with the updated firmware already so you don't need to build it.  You will find this firmware on your machine at this location: "C:/Program Files (x86)/Texas Instruments/SmartRF Tools/BLE Device Monitor/firmware/cc26xx/CC2650SensorTag_BLE_All_v1_20.hex"
